### PR TITLE
check cells and check if it editiable

### PR DESCRIPTION
**Pull request by philippfriberg**
Add Aggregation "cells" for check fields in sap.m.table and check if the field is really
editable (by dynamic readonly/editable).

### CODE REVIEW

File: Validator.js [hunk 1]
```diff
@@ -72,6 +72,16 @@ sap.ui.define([
                     if (oControl.getBinding(aValidateProperties[i])) {
                         // check if a data type exists (which may have validation constraints)
                         if (oControl.getBinding(aValidateProperties[i]).getType()) {
+                        	try {
```
Comment (qualiture) [inline Validator.js]: What is the purpose of the `try/catch` block here? Two questions I have:

 1. In what case would it throw an exception?
 2. What purpose does the `continue` statement serve?

I may be missing something, but I think the added `try/catch` block can be replaced by:

```javascript
editable = !!oControl.getProperty("editable");
```

Comment (qualiture) [reply Validator.js]: As I'm not a big fan of `continue` (and [ESLint will complain as well](https://eslint.org/docs/rules/no-continue) ) could you maybe restructure it to:

```javascript
if (oControl.getBinding(aValidateProperties[i]).getType()) {
    editable = oControl.getProperty("editable");

    if (editable) {
        // try validating the bound value
        try {
            oControlBinding = oControl.getBinding(aValidateProperties[i]);
            oExternalValue = oControl.getProperty(aValidateProperties[i]);
            oInternalValue = oControlBinding.getType().parseValue(oExternalValue, oControlBinding.sInternalType);
            oControlBinding.getType().validateValue(oInternalValue);
        }
        // catch any validation errors
        catch (ex) {
            this._isValid = false;
            oControlBinding = oControl.getBinding(aValidateProperties[i]);
            sap.ui.getCore().getMessageManager().addMessages(
                new Message({
                    message: ex.message,
                    type: MessageType.Error,
                    target: (oControlBinding.getContext() ? oControlBinding.getContext().getPath() + "/" : "") +
                        oControlBinding.getPath(),
                    processor: oControl.getBinding(aValidateProperties[i]).getModel()
                })
            );
        }

        isValidatedControl = true;
    }
}
```

...and see if this works similar for you?

Comment (philippfriberg) [reply Validator.js]: No. I have a Text-Element in the Table too. Now it throw a exception:

Uncaught Error: Property "editable" does not exist in Element sap.m.Text#__text3
    at f.h.getProperty (sap-ui-core.js:471)
    at Validator._validate (/webapp/controller/validator.js?eval:77)

Attribute editable is not in every element available...

This code works:
```
if (oControl.getBinding(aValidateProperties[i]).getType()) {
   try {
       editable = oControl.getProperty("editable");   	
   }
   catch (ex) {
       editable = true;
   }

if (editable) {
```

Comment (qualiture) [reply Validator.js]: Ok, understand 👍 

Can you commit this to the pull request? I will merge then

**Pull request merged**